python/ray/util/placement_group.py

```diff
@@ -1,10 +1,19 @@
 import time
-
-from typing import (List, Dict, Optional, Union)
+from typing import Dict
+from typing import List
+from typing import Optional
+from typing import Union
+from typing import TYPE_CHECKING
 
 import ray
-from ray._raylet import PlacementGroupID, ObjectRef
+from ray._raylet import ObjectRef
+from ray._raylet import PlacementGroupID
 from ray._private.utils import hex_to_binary
+from ray._private.client_mode_hook import client_mode_should_convert
+from ray._private.client_mode_hook import client_mode_wrap
+
+if TYPE_CHECKING:
+    from ray.util.common import ClientObjectRef
```

Comment:
For reference on what these `if TYPE_CHECKING` things are for, see https://stackoverflow.com/questions/39740632/python-type-hinting-without-cyclic-imports

Reply:
Added a # noqa